ext/bcmath/tests/number/operators/short_syntax.phpt

```diff
@@ -0,0 +1,103 @@
+--TEST--
+BcMath\Number operator short syntax
```

Comment:
I think "compound assignment" is the proper term instead of "short syntax"

Reply:
Fixed in https://github.com/php/php-src/pull/13741/commits/0066206b2ecbe7c98d24a0c673ee64f403158e62